components/basics/react-native.js

```diff
@@ -28,9 +28,29 @@ class MyReactNativeComponent extends React.Component {
 const cssToReactNativeSample = (`
 const RotatedBox = styled.View\`
   transform: rotate(90deg);
-  text-shadow-offset: 10 5;
+  text-shadow-offset: 10px 5px;
   font-variant: small-caps;
-  margin: 5 7 2;
+  margin: 5px 7px 2px;
+\`;
+`).trim()
+
+const migrationSample = (`
```

Comment:
Don't need an extensive example if we can write a precise note and link to the codemod :)